### DISCUSSION TITLE
Image loading tutorial for DirectX9 maybe not working. can't build (D3dx9tex.h)

### DISCUSSION THREAD

**Discussion by Zenbeau** [ocornut/imgui#5742] [New Users Build/Link/Run issues ONLY!]
https://github.com/ocornut/imgui/wiki/Image-Loading-and-Displaying-Examples#example-for-directx9-users

I tried to load an image (DirectX9) using this guide, but I get 2-3 errors so I can't build. And I followed the instructions exactly as it said. The errors are on:
#include <D3dx9tex.h>   (error says cannot open source file)
D3DXCreateTextureFromFileA(g_pd3dDevice    (error says identifier is undefined)

I searched google for this D3dx9tex.h but could not find it btw

**Comment by ocornut** [comment]
It is part of D3DX which is a helper toolkit normally installed with DX9 SDK.

**Reply by Zenbeau** [reply]
This DirectX SDK is not supported on Windows 10 (system requirements). (https://www.microsoft.com/en-ca/download/details.aspx?id=6812)
I think only older PCs can access DX9 SDK. Or maybe I made a mistake and overlooked something?

**Comment by ocornut** [comment]
It’s still available for download. However if you can’t use the DX9 sdk why bothering about loading textures for it. Use DX11.
